cli/commands/migrate_apply.go

```diff
@@ -25,7 +25,13 @@ func newMigrateApplyCmd(ec *cli.ExecutionContext) *cobra.Command {
 			return ec.Validate()
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return opts.run()
+			opts.EC.Logger.Info("Applying migrations...")
```

Comment:
@arvi3411301 Need to change these to Spinner, see metadata commands for reference.